README.md

```diff
@@ -29,7 +29,7 @@ In the following, a list of all supported languages with their supported languag
 | [Scheme](http://www.scheme-reports.org)                          |       ? | scheme                | unknown | JavaCC |
 | [EMF Metamodel](https://www.eclipse.org/modeling/emf/)           |  2.25.0 | emf-metamodel         | alpha | EMF |
 | [EMF Metamodel](https://www.eclipse.org/modeling/emf/) (dynamic) |  2.25.0 | emf-metamodel-dynamic | alpha | EMF |
-| Text (naive)                                                     |       - | text                  | legacy | ANTLR |
+| Text (naive)                                                     |       - | text                  | legacy | CoreNLP |
```

Comment:
I'd say we can remove the naive part here with this improvement

Reply:
I don't have a strong opinion on this, but as we don't use any other CoreNLP feature than tokenzing, it's still somewhat naive.

Reply:
I would suggest to keep "naive" until some mechanism like lemma or stemming is added

Reply:
I agree with @dfuchss